src/header/styles.scss

```diff
@@ -171,6 +172,10 @@
     font-weight: awsui.$font-heading-m-weight;
     line-height: awsui.$font-heading-m-line-height;
   }
+
+  @include focus-visible.when-visible {
+    @include styles.link-focus;
```

Comment:
Why is it link-focus and not, say, container focus? I think this one might be better applicable but I don't like the name as it sounds off. Should we maybe introduce a separate one like `header-focus` to ensure those are properly de-coupled?

Reply:
I discussed that with Kashi. Basically, the idea is that we want similar focus behaviour for all text-styled components.

Reply:
ok, sounds good